src/xibo-interactive-control.js

```diff
@@ -412,6 +412,38 @@ window.xiboIC = (function() {
       this.lockPinchZoom(lock);
     },
 
+    /**
+     * Report fault on requested data
+     * @param {Object[]} [params] - Parameters
+     * @param {string} [params.targetId]
+     * @param {string} [params.reason]
+     * @param {boolean} [params.shouldExpire] - Optional expiration
+     * @param {Object[]} [options] - Request options
+     * @param {callback} [options.done]
+     * @param {callback} [options.error]
+     */
+    reportFault(
+      {targetId, reason, shouldExpire = true} = {},
+      {done, error} = {}
+    ) {      // Get target id from the request option or from the global lib var
+      const id = (typeof targetId != 'undefined') ? targetId : _lib.targetId;
+
+      _lib.makeRequest(
+          '/fault',
+          {
+            type: 'POST',
+            data: {
```

Comment:
Sorry Ruben - you've used the article in Rosetta instead of the Issue description - the Issue description is right